src/models/transfer.js

```diff
@@ -116,7 +116,7 @@ PersistentModelMixin(Transfer, sequelize, {
   ledger: Sequelize.STRING(1024),
   debits: JsonField(sequelize, 'Transfer', 'debits'),
   credits: JsonField(sequelize, 'Transfer', 'credits'),
-  part_of_payment: Sequelize.STRING(1024),
+  additional_info: JsonField(sequelize, 'Transfer', 'additional_info'),
```

Comment:
Metadata in IT is "data about data" - hence metadata. For instance file metadata is the size of the data, creation date etc. This field is part of the data, so it's definitely not `metadata` - at a minimum metadata is something that you query separately from the data it describes.

I think the characteristic that we'd want to express is that the ledger is going to ignore whatever you put in this field. So it should be `custom_data` or `external_info` or something like that.

I think `additional_info` is actually not bad, because you should always design APIs from the perspective of the user and the user here is the developer who in her mind just wants to store some "additional fields" on the Transfer. So the name is something very intuitive to her. And of course consistency with Payment Services doesn't hurt.